pkgs/development/python-modules/mdx-truly-sane-lists/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, markdown
+}:
+
+buildPythonPackage rec {
+  pname = "mdx_truly_sane_lists";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "radude";
+    repo = "mdx_truly_sane_lists";
+    rev = version;
+    sha256 = "1h8403ch016cwdy5zklzp7c6xrdyyhl4z07h97qzbafrbq07jyss";
+  };
+
+  propagatedBuildInputs = [ markdown ];
+
+  pythonImportsCheck = [ "mdx_truly_sane_lists" ];
+
+  checkPhase = ''
+    python mdx_truly_sane_lists/tests.py
```

Comment:
```suggestion
    ${python.interpreter} mdx_truly_sane_lists/tests.py
```

Reply:
How it is better if `buildPythonPackage` brings `python` into PATH?

Reply:
Generally you want to use python.interpreter because that works with pypy, too.

Reply:
Ok